schema/proto3/dhcp.proto

```diff
@@ -27,10 +27,11 @@ message DHCPConfiguration {
     uint32 format_version = 1;
     uint32 revision_number = 2;
 
-    string mac_address = 3;
-    string ipv4_address = 4;
-    string ipv6_address = 5;
-    string port_host_name = 6; // for local DNS response
+    string vpc_id = 3;
```

Comment:
Do we need this new field?

Reply:
We don't need it now. But this is suggested by the community which make sense to me. That's why I put it here for future proofing. Let me know if you want to remove this change.

Reply:
will keep this change as discussed.